src/mpermission.cpp

```diff
@@ -32,6 +33,7 @@ const auto TranslationSeparator = QStringLiteral("-");
 const auto SailjailSection = QStringLiteral("X-Sailjail");
 const auto SailjailPermissionsKey = QStringLiteral("Permissions");
 
+#if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
```

Comment:
QStringView added in Qt 5.10 so suppose the new code variant could work there and the requirement here relaxed a bit.

Then again not sure how much it matters to optimize this with refs or views. There shouldn't be that much permissions to handle.